ee/vellum_ee/workflows/display/nodes/vellum/error_node.py

```diff
@@ -20,6 +20,10 @@ class BaseErrorNodeDisplay(BaseNodeDisplay[_ErrorNodeType], Generic[_ErrorNodeTy
 
     __serializable_inputs__ = {ErrorNode.error}
 
+    def serialize_ports(self, display_context: "WorkflowDisplayContext") -> JsonArray:
+        """Error nodes have no ports."""
+        return []
```

Comment:
Instead of defining this here, update the ErrorNode and FinalOutputNode classes themselves with Ports classes with no attributes to show they have no ports

Reply:
OOS an improvement to here on the Graph definition layer